ffcx/codegeneration/ufc.h

```diff
@@ -180,6 +184,9 @@ extern "C"
     /// for a cell (not including global support dofs)
     int num_element_support_dofs;
 
+    /// Return the block size for a VectorElement
+    int submap_block_size;
```

Comment:
What does this mean? Why is it needed? Why is block_size in element, but sub_dofmap_block_size in dofmap?

Reply:
It's needed to handle MixedElements that contain VectorElements. The name is bad: it's used in dolfin to get the blocksize of a subdofmap, but the value itself has nothing to do with sub dofmaps. I'll rename it